source/image-handler/test/test-thumbor-mapping.js

```diff
@@ -47,7 +47,7 @@ describe('process()', function() {
 // parseCustomPath()
 // ----------------------------------------------------------------------------
 describe('parseCustomPath()', function() {
-    describe('001/validPath', function() {
+    /*describe('001/validPath', function() {
```

Comment:
[non-blocking] When you rebase off master, remember to re-add this! I'll update the tests when merging my code so they pass.